src/fipp/visit.cljc

```diff
@@ -1,5 +1,6 @@
 (ns fipp.visit
   "Convert to and visit edn structures."
+  (:refer-clojure :exclude [boolean?])
```

Comment:
That makes sense. I saw on Slack that all of the new core predicates will be added to CLJS soon.